src/test/java/org/reaktivity/specification/amqp/streams/LinkIT.java

```diff
@@ -73,6 +73,18 @@ public void shouldExchangeAttachAsSenderAndReceiver() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Specification({
+        "${scripts}/attach.as.receiver.and.sender/client",
+        "${scripts}/attach.as.receiver.and.sender/server"})
```

Comment:
Suggest renaming `attach.as.receiver.and.sender` to `attach.as.receiver.then.sender`.
Also rename method to `shouldExchangeAttachAsReceiverThenSender`.